types/imtype/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for imtype 1.0.0
+// Project: https://github.com/fatelei/imagetype
+
+export = imType;
+declare namespace imType {
+
+    function isBMP(buffer: Uint8Array): boolean;
+
+    function isJPG(buffer: Uint8Array): boolean;
+
+    function isPNG(buffer: Uint8Array): boolean;
+
+    function isGIF(buffer: Uint8Array): boolean;
+
+    function isTIF(buffer: Uint8Array): boolean;
+
+}
```

Comment:
If you write it like this, it should work.

```suggestion
export function isBMP(buffer: Uint8Array): boolean;

export function isJPG(buffer: Uint8Array): boolean;

export function isPNG(buffer: Uint8Array): boolean;

export function isGIF(buffer: Uint8Array): boolean;

export function isTIF(buffer: Uint8Array): boolean;

```